### ISSUE TITLE
WebGL viewer is broken for IPython3

### ISSUE THREAD

**Issue by rmcgibbo** [mdtraj/mdtraj#751]

**Comment by kyleabeauchamp** [comment]
but wut about jupyter

**Comment by rmcgibbo** [comment]
same thing.

**Comment by rmcgibbo** [comment]
i think the widget interface has been changing between versions. when it came out originally, it was pretty experimental.

**Comment by ctk3b** [comment]
Does it work on python2 for you? Is anyone looking into fixing it already? I get the following on 3.4 and 2.7:
```python
In [1]: import mdtraj as md

In [2]: from mdtraj.html import TrajectoryView, enable_notebook
enable_notebook()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-cc86fc55fde2> in <module>()
      1 from mdtraj.html import TrajectoryView, enable_notebook
----> 2 enable_notebook()

//anaconda/envs/mbuild/lib/python2.7/site-packages/mdtraj-1.3.0.dev0-py2.7-macosx-10.5-x86_64.egg/mdtraj/html/install.pyc in enable_notebook()
     51     libs = ['iview.js','surface.min.js','objexporter.js','filesaver.js','context.js']
     52     fns = [resource_filename('mdtraj', os.path.join('html', 'static', f)) for f in libs]
---> 53     install_nbextension(fns, verbose=0)
     54     display(_REQUIRE_CONFIG)
     55 

//anaconda/envs/mbuild/lib/python2.7/site-packages/IPython/html/nbextensions.pyc in install_nbextension(path, overwrite, symlink, user, prefix, nbextensions_dir, destination, verbose)
    177 
    178     if isinstance(path, (list, tuple)):
--> 179         raise TypeError("path must be a string pointing to a single extension to install; call this function multiple times to install multiple extensions")
    180 
    181     path = cast_unicode_py2(path)

TypeError: path must be a string pointing to a single extension to install; call this function multiple times to install multiple extensions```

**Comment by ctk3b** [comment]
That particular error is actually a very simple fix by looping over the functions. Still can't get it to visualize anything though.

**Comment by golobor** [comment]
I'm having the same issue - even after fixing the line "fns = [resource_filename(...", the widget simply does not show up in the notebook.

**Comment by golobor** [comment]
Hey, 
it seems that the issue was that IPython [changed the way it interacted with require.js] (https://github.com/ipython/ipython/wiki/IPython-3.0-comm-and-widget-migration-document). I found a way to get the widget running, at least on my machine, and submitted [a pull request] (https://github.com/mdtraj/mdtraj/pull/789).

**Comment by rmcgibbo** [comment]
Fixed be @golobor in #789
